GVFS/GVFS.Installer.Windows/Setup.iss

```diff
@@ -646,14 +646,41 @@ begin
   end;
 end;
 
+function IsConfigured(ConfigKey: String): Boolean;
+var
+  ResultCode: integer;
+  ResultString: ansiString;
+begin
+  Result := false
+  if ExecWithResult('gvfs.exe', Format('config %s', [ConfigKey]), '', SW_HIDE, ewWaitUntilTerminated, ResultCode, ResultString) then begin
+    ResultString := AnsiLowercase(Trim(ResultString));
+    Log(Format('IsConfigured(%s): value is %s', [ConfigKey, ResultString]));
+    Result := Length(ResultString) > 1
+  end
+end;
+
+procedure SetIfNotConfigured(ConfigKey: String; ConfigValue: String);
+var
+  ResultCode: integer;
+  ResultString: ansiString;
+begin
+  if IsConfigured(ConfigKey) = false then begin
+    if ExecWithResult('gvfs.exe', Format('config %s %s', [ConfigKey, ConfigValue]), '', SW_HIDE, ewWaitUntilTerminated, ResultCode, ResultString) then begin
+      Log(Format('SetNuGetFeedIfNecessary: Set %s to %s', [ConfigKey, ConfigValue]));
+    end else begin
+      Log(Format('SetNuGetFeedIfNecessary: Failed to set %s with %s', [ConfigKey, SysErrorMessage(ResultCode)]));

```

Comment:
Nit: Here and above `SetNuGetFeedIfNecessary` -> `SetIfNotConfigured`

Reply:
Doh, thanks!